ormar/queryset/queryset.py

```diff
@@ -1127,5 +1132,10 @@ async def bulk_update(  # noqa:  CCR001
         expr = str(expr)
         await self.database.execute_many(expr, ready_objects)
 
-        for objt in objects:
-            objt.set_save_status(True)
+        for obj in objects:
+            obj.set_save_status(True)
+
+        await self.__class__.signals.post_bulk_update.send(
```

Comment:
How to get the signals here?  @collerek

Reply:
You can be explicit and go with `self.model_cls.Meta.signals`, don't remember if it's a property of model or class attribute

Reply:
```
@property
    def signals(self) -> "SignalEmitter":
        """Exposes signals from model Meta"""
        return self.Meta.signals
```

Ok, thats a property of model, so you will get descriptor back from `self.model_cls.signals` and need to go with `self.model_cls.Meta.signals`. Sorry, didn't check before.